src/snowflake/cli/plugins/nativeapp/manager.py

```diff
@@ -715,9 +716,25 @@ def get_validation_result(self, use_scratch_stage: bool):
                     )
 
     def get_events(self) -> list[dict]:
-        if self.account_event_table is None:
+        if not self.account_event_table:
             raise NoEventTableForAccount()
-        return []
+
+        app_name = (
+            unquote_identifier(self.app_name)
+            if is_valid_quoted_identifier(self.app_name)
+            else self.app_name.upper()
```

Comment:
nit: unquote_identifier should already do pretty much all of this, including case normalization